modules/update-upgrading-cli.adoc

```diff
@@ -78,6 +56,18 @@ VERSION IMAGE
 4.10.0  quay.io/openshift-release-dev/ocp-release@sha256:9c5f0df8b192a0d7b46cd5f6a4da2289c155fd5302dec7954f8f06c878160b8b
 ----
 
+. Based on your organization requirements, set the channel to `stable-4.10`, `fast-4.10`, or `eus-4.10`. For more information about the upgrade channels, see xref:../updating/understanding-upgrade-channels-release.adoc#understanding-upgrade-channels-releases[Understanding upgrade channels and releases]:
++
+[source,terminal]
+----
+$ oc adm upgrade channel stable-4.10
+----
++
+[IMPORTANT]
+====
+For production clusters, you must subscribe to a `stable-\*`, `eus-*`, or `fast-*` channel.
```

Comment:
I don't understand why only one of the `*` is escaped, but it seems to render fine, so :shrug:

![image](https://user-images.githubusercontent.com/209920/169085860-b53fe464-bd53-4211-9635-3cee66fefa74.png)

Reply:
@wking Fixed.

Reply:
I see extra `\`.
![0519](https://user-images.githubusercontent.com/20534196/169252664-eb797281-74ce-4d5a-81f9-e6ed495452fa.png)

Reply:
@jiajliu 
i can see the following in .adoc file.
> For production clusters, you must subscribe to a `stable-\*`, `eus-*`, or `fast-*` channel.

If I remove `\`, it won't render in the html. So I'm retaining the format which was already there.

Reply:
ack